### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/openapi-callbacks.md`

### PR DESCRIPTION
**Pull request by devfernandoa**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

**Comment by nazarepiedady** [comment]
@devfernandoa, normally, it is recommended to only modify one single file per pull request, but I think in this one, we could benefit from an exception.

Keep following this path of collaborating with open-source initiatives.

**Comment by nazarepiedady** [comment]
@ceb10n and @cassiobotaro, could you do the kindness of reviewing this pull request?

**Comment by tiangolo** [comment]
@devfernandoa please separate this in 2 PRs, one for each file, this way it will be easier to review. 🤓☕

@nazarepiedady I'll ask you to please don't give instructions about managing the PRs or the repo, those things would be decided by someone in the FastAPI team, and getting external instructions could be confusing for people contributing. Also there's no need to ping others. Thanks!

**Comment by devfernandoa** [comment]
Sorry guys, I forgot to remove the file when switching branches.
Fixed it here, and the other PR is already open as well!

File: docs/pt/docs/advanced/openapi-callbacks.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+Essas faturas terão um `id`, `title` (opcional), `customer` e `total`.
+
+O usuário da sua API (um desenvolvedor externo) criará uma fatura em sua API com uma solicitação POST.
+
+Então sua API irá (vamos imaginar):
+
+* Enviar uma solicitação de pagamento para o desenvolvedor externo.
+* Coletar o dinheiro.
+* Send a notification back to the API user (the external developer).
```
Comment (ceb10n) [inline docs/pt/docs/advanced/openapi-callbacks.md:23]: You've forgotten to remove this line 😄

File: docs/pt/docs/advanced/openapi-callbacks.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+Ele terá uma *operação de rota* que receberá um corpo `Invoice`, e um parâmetro de consulta `callback_url` que conterá a URL para o callback.
+
+Essa parte é bastante normal, a maior parte do código provavelmente já é familiar para você:
+
+```Python hl_lines="9-13  36-53"
+{!../../docs_src/openapi_callbacks/tutorial001.py!}
+```
+
+/// tip | dica
```
Comment (ceb10n) [inline docs/pt/docs/advanced/openapi-callbacks.md:39]: Just a silly suggestion:

We alway start with a capital letter the translation of the admonitions

File: docs/pt/docs/advanced/openapi-callbacks.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+O parâmetro de consulta `callback_url` usa um tipo Pydantic <a href="https://docs.pydantic.dev/latest/api/networks/" class="external-link" target="_blank">Url</a>.
+
+///
+
+The only new thing is the `callbacks=invoices_callback_router.routes` as an argument to the *path operation decorator*. We'll see what that is next.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/openapi-callbacks.md:45]: this line to :D

File: docs/pt/docs/advanced/openapi-callbacks.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+Mas possivelmente a parte mais importante do callback é garantir que o usuário da sua API (o desenvolvedor externo) implemente a *API externa* corretamente, de acordo com os dados que *sua API* vai enviar no corpo da solicitação do callback, etc.
+
+Então, o que faremos a seguir é adicionar o código para documentar como essa *API externa* deve ser para receber o callback de *sua API*.
+
+A documentação aparecerá na interface do Swagger em `/docs` em sua API, e permitirá que os desenvolvedores externos saibam como construir a *API externa*.
+
+Esse exemplo não implementa o callback em si (que poderia ser apenas uma linha de código), apenas a parte da documentação.
+
+/// tip | dica
```
Comment (ceb10n) [inline docs/pt/docs/advanced/openapi-callbacks.md:69]: here to, just start with a capital letter if you want :D

File: docs/pt/docs/advanced/openapi-callbacks.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+
+O callback real é apenas uma solicitação HTTP.
+
+When implementing the callback yourself, you could use something like <a href="https://www.python-httpx.org" class="external-link" target="_blank">HTTPX</a> or <a href="https://requests.readthedocs.io/" class="external-link" target="_blank">Requests</a>.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/openapi-callbacks.md:73]: This line needs to be removed to

File: docs/pt/docs/advanced/openapi-callbacks.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+
+## Escrevendo o código de documentação do callback
+
+Esse código não será executado em seu aplicativo, nós só precisamos dele para *documentar* como essa *API externa* deveria ser.
+
+Mas, você já sabe como criar facilmente documentação automática para uma API com o **FastAPI**.
+
+Então vamos usar esse mesmo conhecimento para documentar como a *API externa* deveria ser... criando as *operações de rota* que a *API externa* deveria implementar (as que sua API irá chamar).
+
+/// tip | dica
```
Comment (ceb10n) [inline docs/pt/docs/advanced/openapi-callbacks.md:86]: here to, just start with a capital letter if you want :D

File: docs/pt/docs/advanced/openapi-callbacks.md [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+
+e esperaria uma resposta daquela *API externa* com um corpo JSON como:
+
+```JSON
+{
+    "ok": true
+}
+```
+
+/// tip | dica
```
Comment (ceb10n) [inline docs/pt/docs/advanced/openapi-callbacks.md:169]: here to, just start with a capital letter if you want :D

File: docs/pt/docs/advanced/openapi-callbacks.md [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+
+Nesse ponto você tem a(s) *operação de rota de callback* necessária(s) (a(s) que o *desenvolvedor externo* deveria implementar na *API externa*) no roteador de callback que você criou acima.
+
+Agora use o parâmetro `callbacks` no decorador da *operação de rota de sua API* para passar o atributo `.routes` (que é na verdade apenas uma `list` de rotas/*operações de rota*) do roteador de callback que você criou acima:
+
+```Python hl_lines="35"
+{!../../docs_src/openapi_callbacks/tutorial001.py!}
+```
+
+/// tip | dica
```
Comment (ceb10n) [inline docs/pt/docs/advanced/openapi-callbacks.md:185]: here to, just start with a capital letter if you want :D

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great work @devfernandoa ✨ 

Just some lines that need to be removed, and some minor suggestions 😄

**Comment by ceb10n** [comment]
Not a so important thing that I've noticed:

The title is incorrect: this file lives under `advanced`, not `how-to`

**Comment by devfernandoa** [comment]
Just fixed these changes, thanks for the review @ceb10n !

**Review by ceb10n — APPROVED** [review]
🤗

**Comment by alejsdev** [comment]
Thank you for being so interested in contributing to FastAPI! @devfernandoa :nerd_face:

**Pull request merged**